lib/dal/src/job/definition/dependent_values_update.rs

```diff
@@ -143,16 +144,19 @@ impl DependentValuesUpdate {
                 let attribute_value_id = attribute_value_id.to_owned(); // release our borrow
 
                 if !seen_ids.contains(&attribute_value_id) {
-                    let join_handle = update_join_set.spawn(
-                        values_from_prototype_function_execution(ctx.clone(), attribute_value_id),
-                    );
-                    task_id_to_av_id.insert(join_handle.id(), attribute_value_id);
+                    let id = Ulid::new();
```

Comment:
@fnichol this is a nit, but I'm wondering if it would be valuable to use the same `Ulid` generator for each loop iteration?

Reply:
A generator to make a new `Ulid`? The only real thing I was after was making sure that each task had a unique identifier across all the tasks so we could return it as part of the task response to make it similar to a unique task id. I could have even been a regular old number (like a `u8`) but I didn't want to do any incrementing bookkeeping 😄 I might be misunderstanding though